src/backend/distributed/connection/connection_management.c

```diff
@@ -10,7 +10,20 @@
 
 #include "postgres.h"
 
+#undef HAVE_POLL
+#undef HAVE_POLL_H
+#undef HAVE_SYS_POLL_H
+
+#ifdef HAVE_POLL_H
 #include <poll.h>
+#endif
+#ifdef HAVE_SYS_POLL_H
```

Comment:
I don't see any `HAVE_SYS_POLL_H` in `pg_config.h.in`. Why are we doing this?

Reply:
Hmm, probably @mtuncer was trying to make this section match [the one here](https://github.com/citusdata/citus/pull/2011/files#diff-b95ac9c2bf85402fbb5eca5edcd94302R35), but that section looks like it was originally broken, we can probably just remove `HAVE_SYS_POLL_H` entirely